src/styles/stopwatch.scss

```diff
@@ -0,0 +1,57 @@
+.stopwatch {
+  display: flex;
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+
+  &__seconds {
+    height: $stopwatch-seconds-height;
+    width: $stopwatch-seconds-width;
+    background-color: $stopwatch-seconds-color;
+    position: absolute;
+    bottom: 40vmin;
+    z-index: 1;
+    animation: rotate-hand $minute-duration linear infinite;
+    transform-origin: bottom center;
+    transform: rotate(0);
+  }
+
+  &__minutes {
+    height: $stopwatch-minutes-height;
+    width: $stopwatch-minutes-width;
+    background-color: $stopwatch-minutes-color;
+    position: absolute;
+    bottom: 40vmin;
```

Comment:
The `bottom` property in `.stopwatch__minutes` should also be half the size of `$stopwatch-size` to center the hand correctly, as per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).